app/helpers/branded-preprint-url.js

```diff
@@ -0,0 +1,25 @@
+import Ember from 'ember';
+import pathJoin from '../utils/path-join';
+
+/**
+ * @module ember-preprints
+ * @submodule helpers
+ */
+
+/**
+ * To determine the link url for branded preprints on the provider carousel
+ *
+ * @class route-prefix
+ */
+
+export function brandedPreprintUrl(provider, lightLogo) {
```

Comment:
The canonical way to use parameters in a helper is:
```
export function myHelper(params, hash) {
    let [param1, param2] = params;
    let myNamedParam = hash.foo;
}
```
So `params` is a list of positional parameters and `hash` is an object containing the named parameters.